<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="create user in pet shop" tests="1" skipped="0" failures="1" errors="0" timestamp="2023-03-22T16:25:05" hostname="LAPTOP-ME7AF93M" time="8.528">
  <properties/>
  <testcase name="Validate user creation" classname="create user in pet shop" time="8.528">
    <failure message="java.lang.AssertionError: 1 expectation failed.&#10;Expected status code &lt;200&gt; but was &lt;500&gt;.&#10;" type="java.lang.AssertionError">java.lang.AssertionError: 1 expectation failed.
Expected status code &lt;200&gt; but was &lt;500&gt;.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:60)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:58)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:46)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:50)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:50)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at io.swagger.petstore.definitions.CreateUserDefinitions.verifyUserCreationWithStatus(CreateUserDefinitions.java:26)
	at âœ½.verify user creation with status 200(file:///D:/Automatizacion/retoServicios/src/test/resources/create_user.feature:14)
</failure>
  </testcase>
  <system-out><![CDATA[{"id":23,"username":"Juan23","firstName":"Camilo","lastName":"Penuela","email":"juanC1@yopmail.com","password":"juan123","phone":"30012345678","userStatus":0}
]]></system-out>
  <system-err><![CDATA[[Test worker] INFO io.cucumber.core.plugin.SerenityReporter - Running feature from file:///D:/Automatizacion/retoServicios/src/test/resources/create_user.feature
[Test worker] WARN io.cucumber.core.plugin.SerenityReporter - Feature from file:///D:/Automatizacion/retoServicios/src/test/resources/create_user.feature is not under the 'features' directory. Requirements report will not be correctly generated!
[Test worker] INFO  - 
  _____   ___   ___   _____     ___   _____     _     ___   _____   ___   ___  
 |_   _| | __| / __| |_   _|   / __| |_   _|   /_\   | _ \ |_   _| | __| |   \ 
   | |   | _|  \__ \   | |     \__ \   | |    / _ \  |   /   | |   | _|  | |) |
   |_|   |___| |___/   |_|     |___/   |_|   /_/ \_\ |_|_\   |_|   |___| |___/ 

Validate user creation(create-user-in-pet-shop;validate-user-creation)
--------------------------------------------------------------------------------
[Test worker] ERROR  - 
  _____   ___   ___   _____     ___     _     ___   _      ___   ___  
 |_   _| | __| / __| |_   _|   | __|   /_\   |_ _| | |    | __| |   \ 
   | |   | _|  \__ \   | |     | _|   / _ \   | |  | |__  | _|  | |) |
   |_|   |___| |___/   |_|     |_|   /_/ \_\ |___| |____| |___| |___/ 

Validate user creation
-----------------------------------------------------------------------
[Test worker] ERROR  -     Test failed at step: Then verify user creation with status 200
[Test worker] ERROR  -     1 expectation failed.Expected status code <200> but was <500>.
]]></system-err>
</testsuite>
